chardet/sbcharsetprober.py

```diff
@@ -36,7 +36,7 @@ class SingleByteCharSetProber(CharSetProber):
     SB_ENOUGH_REL_THRESHOLD = 1024
     POSITIVE_SHORTCUT_THRESHOLD = 0.95
     NEGATIVE_SHORTCUT_THRESHOLD = 0.05
-    SYMBOL_CAT_ORDER = 250
+    SYMBOL_CAT_ORDER = 254
```

Comment:
This was wrong before, but I think it should actually be 253, since the old comment said:

```
# 255: Control characters that usually does not exist in any text
# 254: Carriage/Return
# 253: symbol (punctuation) that does not belong to word
# 252: 0 - 9
```